test/unit/io/sequence_file/sequence_file_format_genbank_test.cpp

```diff
@@ -0,0 +1,452 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+#include <sstream>
+
+#include <gtest/gtest.h>
+
+#include <seqan3/alphabet/quality/all.hpp>
+#include <seqan3/io/sequence_file/input.hpp>
+#include <seqan3/io/sequence_file/input_format_concept.hpp>
+#include <seqan3/io/sequence_file/output.hpp>
+#include <seqan3/io/sequence_file/output_format_concept.hpp>
+#include <seqan3/io/sequence_file/format_genbank.hpp>
+#include <seqan3/test/pretty_printing.hpp>
+
+using namespace seqan3;
+
+// ----------------------------------------------------------------------------
+// general
+// ----------------------------------------------------------------------------
+
+TEST(general, concepts)
+{
+    EXPECT_TRUE((SequenceFileInputFormat<sequence_file_format_genbank>));
+    EXPECT_TRUE((SequenceFileOutputFormat<sequence_file_format_genbank>));
+}
+
+// ----------------------------------------------------------------------------
+// reading
+// ----------------------------------------------------------------------------
+
+struct read : public ::testing::Test
+{
+    std::vector<std::string> expected_ids
+    {
+        { "ID1" },
+        { "ID2" },
+        { "ID3" },
+    };
+
+    std::vector<dna5_vector> expected_seqs
+    {
+        { "ACGTTTTTTTTTTTTTTT"_dna5 },
+        { "ACGTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT"_dna5 },
+        { "ACGTTTA"_dna5 },
+    };
+
+	std::string input
```

Comment:
```suggestion
    std::string input
```

No tabs in the file! Please double check by search for the tab-character!